src/main/java/net/dv8tion/jda/core/audio/AudioWebSocket.java

```diff
@@ -521,7 +517,7 @@ private void setupKeepAlive(final int keepAliveInterval)
 
         try
         {
-            keepAlivePool.scheduleAtFixedRate(keepAliveRunnable, 0, keepAliveInterval, TimeUnit.MILLISECONDS);
+            keepAliveHandle = keepAlivePool.scheduleAtFixedRate(keepAliveRunnable, 0, keepAliveInterval, TimeUnit.MILLISECONDS);
```

Comment:
like he said, setupKeepAlive doesn't fire twice, and if it did we would have a lovely FATAL log message to tell us it happened. Should be fine